src/test/java/guitests/FilterTextFieldTest.java

```diff
@@ -121,6 +62,96 @@ public void cancelTest() {
         awaitCondition(() -> toggle.get() % 2 == 1);
     }
 
+    @Test
+    public void filterTextFieldColor_validFilter_validFilterStyleApplied(){
+        FilterTextField field = getFirstPanelField();
+
+        clearField();
+        type("is:open");
+        assertTrue(field.getStyle().contains(VALID_FILTER_STYLE));
```

Comment:
Maybe we can factor out the clear, type, assert pattern here, with a function that just takes the string to test.